### ISSUE TITLE
Bad status and size when backend not found

### ISSUE THREAD

**Issue by maxwo** [containous/traefik#2112]
### Do you want to request a *feature* or report a *bug*?

Bug on the accesslog file

### What did you do?

When requesting an undiscovered backend, status and size informations are not present in accesslog.

### What did you expect to see?

Traefik internal status and size, eg. "404".

### What did you see instead?

```
172.20.48.154 - - [12/Sep/2017:15:10:41 +0000] "GET /prometheus/alerts HTTP/1.1" %!d(<nil>) %!d(<nil>) - "Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/60.0.3112.113 Safari/537.36" 155025 - - 0ms
```

### Output of `traefik version`: (_What version of Traefik are you using?_)

```
Version:      v1.4.0-rc1
Codename:     roquefort
Go version:   go1.9
Built:        2017-08-29_08:16:35AM
OS/Arch:      linux/amd64
```

### What is your environment & configuration (arguments, toml, provider, platform, ...)?

```json
{
 "GraceTimeOut": 10000000000,
 "Debug": false,
 "CheckNewVersion": true,
 "AccessLogsFile": "/var/log/traefik/access.log",
 "AccessLog": null,
 "TraefikLogsFile": "/var/log/traefik/traefik.log",
 "LogLevel": "WARN",
 "EntryPoints": {
  "http": {
   "Network": "",
   "Address": ":1080",
   "TLS": null,
   "Redirect": null,
   "Auth": null,
   "WhitelistSourceRange": null,
   "Compress": false,
   "ProxyProtocol": false
  },
  "https": {
   "Network": "",
   "Address": ":1443",
   "TLS": null,
   "Redirect": null,
   "Auth": null,
   "WhitelistSourceRange": null,
   "Compress": false,
   "ProxyProtocol": false
  }
 },
 "Cluster": null,
 "Constraints": [],
 "ACME": null,
 "DefaultEntryPoints": [],
 "ProvidersThrottleDuration": 2000000000,
 "MaxIdleConnsPerHost": 200,
 "IdleTimeout": 0,
 "InsecureSkipVerify": false,
 "RootCAs": null,
 "Retry": null,
 "HealthCheck": {
  "Interval": 30000000000
 },
 "RespondingTimeouts": {
  "ReadTimeout": 0,
  "WriteTimeout": 0,
  "IdleTimeout": 180000000000
 },
 "ForwardingTimeouts": {
  "DialTimeout": 30000000000,
  "ResponseHeaderTimeout": 0
 },
 "Docker": null,
 "File": null,
 "Web": {
  "Address": ":8080",
  "CertFile": "",
  "KeyFile": "",
  "ReadOnly": false,
  "Statistics": null,
  "Metrics": {
   "Prometheus": {
    "Buckets": [
     0.1,
     0.3,
     1.2,
     5
    ]
   },
   "Datadog": null,
   "StatsD": null
  },
  "Path": "",
  "Auth": null,
  "Debug": false,
  "CurrentConfigurations": null,
  "Stats": null,
  "StatsRecorder": null
 },
 "Marathon": null,
 "Consul": null,
 "ConsulCatalog": null,
 "Etcd": null,
 "Zookeeper": null,
 "Boltdb": null,
 "Kubernetes": null,
 "Mesos": null,
 "Eureka": null,
 "ECS": null,
 "Rancher": {
  "Watch": true,
  "Filename": "",
  "Constraints": [
   "tag==control"
  ],
  "Trace": false,
  "DebugLogGeneratedTemplate": false,
  "Endpoint": "",
  "AccessKey": "",
  "SecretKey": "",
  "API": null,
  "Metadata": {
   "IntervalPoll": false,
   "Prefix": ""
  },
  "Domain": "",
  "RefreshSeconds": 1,
  "ExposedByDefault": true,
  "EnableServiceHealthFilter": false
 },
 "DynamoDB": null,
 "ConfigFile": "/etc/traefik/traefik.toml"
}
```

**Comment by ldez** [comment]
Duplicate of 2055, #2096

**Comment by ldez** [comment]
It's fixed in 1.4.0-rc2.
